test/integration-orientdb/tests/config/database.test.js

```diff
@@ -92,7 +94,24 @@ describe('Config tests', function() {
           });
         });
       });
-      
+
+			it('should be able to connect with database credentials only', function(done) {
+				self.waterline.teardown(function (err) {
+					if (err) { return done(err); }
+					
+					var newConfig = _.cloneDeep(configConn);
+					newConfig.user = undefined;
+					newConfig.password = undefined;
+					newConfig.options.databaseUser = newConfig.databaseTestUser;
```

Comment:
`databaseTestUser` and `databaseTestPassword` are not on [test-connection.json](https://github.com/appscot/sails-orientdb/blob/master/test/test-connection.json). Can you please use the default `admin`/`admin` to avoid sending `undefined`?

And please fix the indentation.